packages/contract/ai/super-admin-add-model.request.dto.ts

```diff
@@ -0,0 +1,7 @@
+import { z } from "zod";
+
+export const SuperAdminAddModelRequestSchema = z.object({
+  chatLlmName: z.string(),
+  modelName: z.string(),
+  apiKey: z.string(),
+});
```

Comment:
On the creation, not all fields are mandatory, or are they? If they aren't, they should be flagged as optional, if not, all good as is 👍

Reply:
After some discussion we refactor the schema so now it should be ok.